lib/instrumentation/core/http.js

```diff
@@ -71,6 +71,13 @@ function wrapEmitWithTransaction(agent, emit, isHTTPS) {
     const segment = tracer.createSegment(request.url, recordWeb)
     segment.start()
 
+    if (request.method != null) {
```

Comment:
Looks like its added to transaction.verb, used in transaction name-state, but not added as an attribute.  I added it because it was listed as a Transaction Agent Attribute in the Attribute Catalog.  We do add it as a Transaction Attribute in our AWS instrumentation `setWebRequest` function.  Maybe I need to add it to Transaction explicitly here?

Reply:
interesting. i'm not sure. i'm surprised we'd have for lambda and not here unless it originated around that time.